audio.js

```diff
@@ -97,7 +110,7 @@ class Audio {
     } else {
       // extract from fall back audio
       that.fallBackAudio[0].onplay = function() {
-        var interval = setInterval(that.extractMediaFeature, 30);
+        var interval = setInterval(that.extractMediaFeature, 23.3);
```

Comment:
Where does 23.3 come from?

Reply:
Our buffer size is 1024. 23.3 is 1024/(441*3).